deps/verifier/src/tdx/quote.rs

```diff
@@ -506,7 +509,65 @@ pub async fn ecdsa_quote_verification(quote: &[u8]) -> Result<()> {
         }
     }
 
-    Ok(())
+    let tcb_status_desc = SgxQlQvResultWrapper(quote_verification_result).as_str();
+
+    let advisory_ids: Vec<Value> = get_sa_list(&supp_data.sa_list)
+        .iter()
+        .map(|s| Value::String(s.to_string()))
+        .collect();
+
+    let mut claims_map = Map::new();
+
+    claims_map.insert(
+        "tcb_status".to_string(),
+        Value::String(tcb_status_desc.to_string()),
+    );
+    claims_map.insert(
+        "collateral_expiration_status".to_string(),
+        Value::String(collateral_expiration_status.to_string()),
+    );
+    claims_map.insert("advisory_ids".to_string(), Value::Array(advisory_ids));
+
+    Ok(claims_map)
```

Comment:
we could move the new code to the match arm that covers OK and the non-terminal results and return from there, wdyt?